[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python3-zope.security",
	"source": "zope.security (4.0.3-2)",
	"version": "4.0.3-2+b3",
	"section": "zope",
	"priority": "optional",
	"architecture": "amd64",
	"depends": [
		"python3 (<< 3.8)",
		"python3 (>= 3.6~)",
		"python3-pkg-resources",
		"python3-zope.component",
		"python3-zope.i18nmessageid",
		"python3-zope.interface",
		"python3-zope.location",
		"python3-zope.proxy",
		"python3-zope.schema",
		"python3:any (>= 3.3.2-2~)",
		"libc6 (>= 2.4)"
	],
	"conflicts": [
		"zope3"
	],
	"installedSize": 614,
	"maintainer": "Debian/Ubuntu Zope Team <pkg-zope-developers@lists.alioth.debian.org>",
	"description": "Zope Security Framework",
	"homepage": "http://pypi.python.org/pypi/zope.security",
	"filename": "pool/main/z/zope.security/python3-zope.security_4.0.3-2+b3_amd64.deb",
	"size": 79392,
	"md5": "7a2bea21103f2778b5bb740bcde7a927",
	"sha256": "404e2cfe1168912ee5e02fc376e1452bd241bed59ee11e90fbe4a0fa89969281",
	"descriptionMd5": "e9f54e364b11459c8fb9ffe7642eb3bf"
}]